rust/src/wallet.rs

```diff
@@ -89,6 +94,16 @@ pub fn get_balance() -> Result<bdk::Balance> {
         .sync()
 }
 
+pub fn get_seed_phrase() -> Result<Vec<String>> {
+    let wallet = WALLET
+        .try_get()
+        .context("Wallet uninitialised")?
+        .lock()
+        .map_err(|_| anyhow!("cannot acquire wallet lock"))?;
```

Comment:
🛠 might be a good time to extract this boilerplate into a free function

Reply:
https://github.com/itchysats/10101/pull/102/commits/510d5d4eed1bdcfd082f8217f3b0e0c1e918c0c5